traveller/static/themes/front/conftheme/2021/parts/talk_actions.html

```diff
@@ -31,6 +31,11 @@
 <div class="paper container">
 
   <label class="paper-btn btn-danger btn-small" for="modal-1">Delete</label>
+  <label class="paper-btn btn-primary btn-small" style="float:right; padding: 10px 15px;">
+    <a href="{{ url_for('y.profile', year=2021) }}" style="text-decoration:none; background-image: none; font-size:12pt;">
```

Comment:
I am pretty sure that hardcoding the year is not a good idea in general.

But for now, and especially for this year's FlaskCon this will work out.

Reply:
@jugmac00 it's good now as for 2022, there will be another folder in conftheme. Like those html files are only for 2021 so hardcoding is not a problem